README.md

```diff
@@ -0,0 +1,9 @@
+# LLVM IR Language Suppport
```

Comment:
```suggestion
# LLVM IR Language Support
```

Can we have build and install instructions for dumb people like me?

Reply:
Typo fixed
Added `COMPILE.md` with compilation instructions